lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -261,28 +266,9 @@ proc fileOptionsTabPage(string $tabLayout)
             -af nodeNamePrefix top 0
             -ac nodeNamePrefix left 3 nameClash2OptionMenu
         nameClashForm;
-    
-    // Trying to concatenate uiRes() strings in a single statement generates a
-    // syntax error.
-    string $prefixesMsg = `uiRes("m_fileOptions.kPrefixesMsg1")` +
-        `uiRes("m_fileOptions.kPrefixesMsg2")` +
-        `uiRes("m_fileOptions.kPrefixesMsg3")`;
-
-    text
-        //-align "center" 
-        -label $prefixesMsg
-        prefixesObsoleteLabel;
-
-    string $namespaceOptionVar = "referenceUseNamespacesDuringFileIO";
-    int $useNS = (`optionVar -query $namespaceOptionVar`);
-    if ($useNS)
-    {
-        text -e -visible false prefixesObsoleteLabel;
-    }
-    else
-    {
-        text -e -visible true prefixesObsoleteLabel;
-    }
+
+    // Need this text field as it is referenced by Maya callbacks.
+    text -visible false prefixesObsoleteLabel;
```

Comment:
Here we only need the text field (not the actual text in it) since it will always be hidden.